polylogue/ui/facade.py

```
@@ -177,15 +183,102 @@ def __post_init__(self) -> None:
+    _ensure_interactive_deps()
+    return GumConsoleFacade(plain=False)
+
+
+_REQUIRED_INTERACTIVE_CMDS = ("gum", "sk", "bat", "glow", "delta")
+
+
+def _ensure_interactive_deps() -> None:
+    missing: List[str] = []
+    for cmd in _REQUIRED_INTERACTIVE_CMDS:
+        if shutil.which(cmd) is None:
+            missing.append(cmd)
```

Ruff (0.15.9)

[warning] Use a list comprehension to create a transformed list

(PERF401)